sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -509,6 +526,11 @@ object ScalaNativePluginInternal {
     files.filterNot(f => isOtherGC(f.getPath().toString, nativelib))
   }
 
+  private val isWindows: Boolean = {
+    val os = Option(sys props "os.name").getOrElse("")
```

Comment:
It should always be set. If `getProperty()` returns `null`, it would be better to throw an exception to prevent any unexpected behaviour further down the line.